package.json

```diff
@@ -130,7 +130,7 @@
     "handlebars-helpers": "^0.10.0",
     "jsdom": "^16.6.0",
     "jsdom-global": "^3.0.2",
-    "mocha": "^8.4.0",
+    "mocha": "^6.2.0",
```

Comment:
Downgraded this because it was giving me `npm install` errors with the most current version on `npm` v7 — now that I've downgraded to `npm` v6 this may not be necessary, but I've left it